cli/sawtooth_cli/admin.py

```diff
@@ -22,15 +21,19 @@
 from sawtooth_cli.admin_command.utils import ensure_directory
 
 
-def do_admin(args):
+def do_admin(args, parser):
     data_dir = ensure_directory('data', '/var/lib/sawtooth')
 
+    if args.admin_cmd is None:
```

Comment:
@vaporos Should I close this PR, so that we can consider this issue more?
My changes prevent a bunch of exceptions or errors when subcommands are left out. Maybe some of the changes can be implemented, or this should be a new Jira that involves discussion and then a more  considered change.